### ISSUE TITLE
Empty table header in APD Builder

### ISSUE THREAD

**Issue by jeromeleecms** [18F/cms-hitech-apd#1780]
Per Wave, there are multiple `<th>` that contains no text. 

Rationale: 

*Why It Matters*
The `<th>` element helps associate table cells with the correct row/column headers. A `<th>` that contains no text may result in cells with missing or incorrect header information.
*How to Fix It*
If the table cell is a header, provide text within the cell that describes the column or row. If the cell is not a header or must remain empty (such as the top-left cell in a data table), make the cell a `<td>` rather than a `<th>`.
*The Algorithm... in English*
A `<th>` element does not contain any text (or contains only spaces) and no images with alternative text.

### This task is done when...
 - [ ] This error is no longer reported in WAVE (or comparable testing device)
 - [ ] Someone else has looked at the code

**Comment by mgwalker** [comment]
Just double-checked and there's only one place where switching to `<td>` might not be the solution. The header row above the quarterly federal share tables in the proposed budget section has two cells, but one of them is empty. Maybe it could get a `colspan`?

![screenshot showing quarterly federal share table](https://user-images.githubusercontent.com/1775733/65061063-bc306a00-d93e-11e9-99cd-c505136fb12f.png)
